docs/benchmarks/micro-benchmarks.md

```diff
@@ -26,18 +26,14 @@ id: micro-benchmarks
         <b>Micro Benchmark</b>
         <img src={require('../assets/bar.png').default}/>
       </td>
-      <td>
-        <b>Model Benchmark</b>
-        <img src={require('../assets/bar.png').default}/>
-      </td>
     </tr>
     <tr valign="top">
       <td align="center" valign="middle">
         <b>Metrics</b>
       </td>
       <td>
         <ul><li><b>Computation Benchmark</b></li>
```

Comment:
Move "Memory benchmark" into section "Communication Benchmark"? and there have two "H2D_Mem_BW_", pls change one to "D2H_Mem_BW_".

Reply:
moved and fixed